pkg/pillar/volumehandlers/nokube.go

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 202 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build !kubevirt
+
+package volumehandlers
+
+// NewCSIHandler in this file is just stub for non-kubevirt hypervisors.
+func NewCSIHandler(common commonVolumeHandler, useVHost bool) VolumeHandler {
+	panic("Kubernetes CSI handler is not built")
```

Comment:
Why Panic ?  Does not it cause device reboots unnecessarily ? May be just error out ?

Reply:
We already use panic/log.Fatal in EVE for situations where it does not make sense to continue. I wanted to preserve this approach. It does lead to reboots but at least user notices the problem immediately.

Reply:
Yeah but does that cause continuous device reboots ? How to stop it ?

Reply:
This is how we have watchdog configured, to reboot when zedbox goes down.

If Kubernetes deps are not built and EVE tries to use them, sooner or later it will hit some nil-dereference and panic. So there is no way to avoid it. I would rather throw our own panic with readable error message.